src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
If we're removing that line we might as well remove the entire `registerDrivers` method to avoid auto-registering any of these drivers. That's the direction I think I'd like to head -- but if it's ok with you, let's leave it for another PR since it's a backwards-incompatible change.